Documentation/learning/design-learner.md

```diff
@@ -1,5 +1,7 @@
 ---
 title: etcd learner design
+weight: 3375
+description: Common challenges with membership reconfiguration
```

Comment:
I would suggest to change to "Mitigating common challenges of membership reconfiguration" as learner solves the challenges described at the top of the doc. Thanks!

Reply:
Updated